example/Podfile.lock

```diff
@@ -110,7 +110,7 @@ DEPENDENCIES:
   - React-RCTText (from `node_modules/react-native/Libraries/Text`)
   - React-RCTVibration (from `node_modules/react-native/Libraries/Vibration`)
   - React-RCTWebSocket (from `node_modules/react-native/Libraries/WebSocket`)
-  - ReactTestApp-DevSupport (from `./..`)
+  - ReactTestApp-DevSupport (from `./node_modules/react-native-test-app`)
```

Comment:
You should use `yarn link` as described in the README. That will revert this line and the line below.